src/widget/xy_pad.rs

```diff
@@ -237,9 +238,8 @@ impl<'a, X, Y, F> Widget for XYPad<'a, X, Y, F>
 
         let value_or_bounds_have_changed = {
             let v = state.view();
-            v.x != x || v.y != y
-                || v.min_x != min_x || v.max_x != max_x
-                || v.min_y != min_y || v.max_y != max_y
+            v.x != x || v.y != y || v.min_x != min_x || v.max_x != max_x || v.min_y != min_y ||
+            v.max_y != max_y
```

Comment:
I prefer the symmetry of the original formatting here, not sure what we can do about it with rustfmt config though.